### PR TITLE
feat(docs): add controls and essentials

### PR DESCRIPTION
**Pull request by viktoria-schwarz**
#926

### CODE REVIEW

File: packages/components/src/Field/Field.stories.mdx [hunk 1]
```diff
@@ -1,57 +1,64 @@
...
+    },
+    label: {
+      control: {
+        type: 'text',
+      },
+      type: { required: true },
+    },
+    error: {
+      control: {
+        type: 'boolean',
```
Comment (ti10le) [inline packages/components/src/Field/Field.stories.mdx:31]: Prop error is type string at the moment and changing some props here has no effect.

File: packages/components/src/Select/Select.stories.mdx [hunk 2]
```diff
@@ -1,51 +1,69 @@
...
+      options: [true, false],
+      table: {
+        defaultValue: {
+          summary: false,
+        },
+      },
+    },
+    error: {
+      control: {
+        type: 'boolean',
```
Comment (ti10le) [inline packages/components/src/Select/Select.stories.mdx:42]: string because of the error message

**Review by ti10le — CHANGES_REQUESTED** [review]

**Comment by ti10le** [comment]
When you click show code in storybook you got <object object> if you used our <Text> component. Storybook doesnt have the import correctly.

**Comment by viktoria-schwarz** [comment]
> When you click show code in storybook you got if you used our component. Storybook doesnt have the import correctly.

I think it happens each time a Polymorphic Component is used (storybook doesn't know early enough what to render, so it just outputs object object), but I would leave it for now and note it down for later

**Review by ti10le — APPROVED** [review]

**Pull request merged**